packages/discovery/src/discovery/output/toDiscoveryOutput.test.ts

```diff
@@ -14,6 +14,7 @@ describe(processAnalysis.name, () => {
     values: {},
     isVerified: true,
     deploymentTimestamp: new UnixTime(1234),
+    deploymentBlockNumber: 9876,
```

Comment:
We fetch this info, but when will we start to write it to the discovered.json? Currently `toDiscoveryOutput` ignores this variable

Reply:
This can be done in a separate story. I want this one to cause as little impact as possible.